sdk/ml/azure-ai-ml/azure/ai/ml/operations/_model_operations.py

```diff
@@ -436,33 +440,24 @@ def list(
             )
         )
 
-    def _get_latest_version(self, name: str) -> Model:
-        """Returns the latest version of the asset with the given name.
+    @monitor_with_activity(logger, "Model.Share", ActivityType.PUBLICAPI)
+    def share(self, name, version, share_with_name, share_with_version, registry_name) -> Model:
+        """Share a model asset from workspace to registry.
 
-        Latest is defined as the most recently created, not the most recently updated.
-        """
-        result = _get_latest(
-            name,
-            self._model_versions_operation,
-            self._resource_group_name,
-            self._workspace_name,
-        )
-        return Model._from_rest_object(result)
-
-    # pylint: disable=no-self-use
-    def _prepare_to_copy(
-        self, model: Model, name: Optional[str] = None, version: Optional[str] = None
-    ) -> WorkspaceAssetReference:
```

Comment:
Are we allowed to just delete this?
Or do we need to mark as deprecated instead

Reply:
I don't think we should keep it, if someone still wants they can use the existing SDK, and PMs are anyway updating the DOCs.., are we keeping different versins of DOCS?

Reply:
I'm all for deleting, but just wanted to make sure we don't do anything we're not allowed to. It's `_` prefix but not sure if we have any other Azure rule

Reply:
This should not affect any Azure rule per my understanding, we keep changing private methods...as long as this is not exposed publicly